doc/nrf/ug_bt_mesh_fota.rst

```diff
@@ -0,0 +1,66 @@
+.. _ug_bt_mesh_fota:
+
+Performing FOTA updates in Bluetooth mesh
+#########################################
+
+In |NCS|, there is currently no support for Device Firmware Update (DFU) over Bluetooth® mesh when performing firmware over-the-air (FOTA) updates for your Bluetooth mesh devices and applications.
+However, support for FOTA updates with point-to-point DFU over Bluetooth Low Energy using the MCUmgr subsystem and the Simple Management Protocol (SMP) is available.
+
+Following the instructions described in :ref:`FOTA over Bluetooth Low Energy<ug_nrf52_developing_ble_fota>`, you can enable the support for and perform FOTA updates using a mobile app.
+
+The :ref:`bluetooth_mesh_light` sample enables support for point-to-point DFU over Bluetooth Low Energy for nRF52 Series development kits using a configuration overlay file :file:`overlay-dfu.conf`.
+FOTA update support is enabled by you simply building the sample with this configuration overlay file and flashing it to your development kit, and you can go straight to performing the FOTA update and itiating the DFU process on your device.
+Alternatively, see :ref:`FOTA update sample<ug_nrf52_developing_ble_fota>` for details on how to use the :ref:`zephyr:smp_svr_sample` to set up your project to support FOTA updates in |NCS|.
+
+Point-to-point DFU over Bluetooth Low Energy for :ref:`zephyr:thingy53_nrf5340` is supported by default, out-of-the-box.
+See :ref:`thingy53_app_update` for more information about updating firmware image on :ref:`zephyr:thingy53_nrf5340`.
+
+.. note::
+      When performing FOTA update on a Bluetooth mesh device and the device's composition data is going to change after the update, unprovision the device before downloading the new image.
```

Comment:
```suggestion
      When performing FOTA update on a Bluetooth® mesh device, the device's composition data is going to change after the update. Unprovision the device before downloading the new image.
```

Reply:
Will device's composition data always change after the update?

Reply:
No, not necessarily. But we do not know what happens if you download a new sample with old composition data from a previous sample. I do not think it is necessary if the updated sample has the same composition data layout.

Reply:
Talked about this with @PavelVPV, and basically this should work... And if not, then this is a bug in his opinion, and it should be fixed.
Let me know if this needs more discussion :)